.clang-format

```
@@ -0,0 +1,123 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
```

```suggestion
BasedOnStyle:  Google
```
Currently we're using code style based on Google, if we don't really know what it does, considering merge this to dev branch.

---

Let me try to modify it again and see how it works